tests/errors/semantic/blocking/ADD_SET_UNHASHABLE_TYPE_2.py

```diff
@@ -0,0 +1,6 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring
+
+a = {(0,7),(4,2), (9,6)}
+b = 4
+a.add(b)
```

Comment:
In this case, the error will be raised on line 3.

Reply:
What is the point of lines 4 and 5 then? They are just confusing and I suggest to remove them.

Reply:
what about this example :
```suggestion
a = {4, 7, 9}
b = {3,5}
a.add(b)
```
the point is to test the add method when adding an unhashable type.

Reply:
In your original code you were not testing the add method, because an error was raised on line 3 when you constructed the set.

The point is **not** testing the add method when adding an unhashable type, because that makes really no sense. Let me explain:

1. Pyccel can only create sets from hashable types (or at least some specific hashable types). If you pass an unhashable type to `{}` an error is raised.
2. When using `add` you must pass a single object of the same type as the objects contained in the set. If you pass the wrong type a `TypeError` should be raised. It does not matter if the passed type is hashable or not. There is one correct type and all the others are wrong.

Do you understand what I mean?

Reply:
fixed : e441c23